src/MessageTrait.php

```diff
@@ -140,6 +140,7 @@ public function withBody(StreamInterface $body): self
     private function setHeaders(array $headers): void
     {
         foreach ($headers as $header => $value) {
+            if (is_int($header)) $header = strval($header);
```

Comment:
This one-liner needs to be changed to actually match the project’s style guide. Running `php-cs-fixer` gives us:

```suggestion
            if (\is_int($header)) {
                $header = (string) $header;
            }
```